ortools/sat/csharp/IntegerExpressions.cs

```diff
@@ -69,6 +69,21 @@ public static void AddOrIncrement(this Dictionary<IntVar, long> dict, IntVar key
             list.Capacity = Math.Max(list.Count + collection.Count, list.Capacity);
         }
     }
+
+#if NETFRAMEWORK
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Not sure adding a method is worth it if it is called only once.

Reply:
The plan would be to use it soon in another place, next PR, this is why I added it here.